ch1.py

```diff
@@ -0,0 +1,43 @@
+import requests
+ 
+
+url = f'https://www.theverge.com/rss/index.xml'
+r= requests.get(url)
+
+if r.status_code == 200:
+	xml_content = r.text
+	print(xml_content)
+else:
+	print("error")
```

Comment:
Aquí no ocupa imprimir el contenido de la pagina si está checando el status code.
```suggestion
if r.status_code != 200:
	print(f"error with status code {r.status_code}")
```